doc/man3/EVP_DigestInit.pod

```diff
@@ -226,7 +226,7 @@ few bytes.
 =item EVP_DigestInit()
 
 Behaves in the same way as EVP_DigestInit_ex() except it always uses the
-default digest implementation.
+default digest implementation and clears all flags.
```

Comment:
Might it be better to instead mention `and calls EVP_MD_CTX_reset`?
A little more is done that clearing the flags.